evennia/contrib/rpg/rpsystem/tests.py

```diff
@@ -212,18 +178,18 @@ def test_send_emote(self):
         rpsystem.send_emote(speaker, receivers, emote, case_sensitive=False)
         self.assertEqual(
             self.out0,
-            "With a flair, |bSender|n looks at |bThe first receiver of emotes.|n "
+            "With a flair, |mSender|n looks at |bThe first receiver of emotes.|n "
```

Comment:
Is there a purpose to changing this (apart from you preferring one color over the other :wink: )

Reply:
It's intended to highlight the difference between a recog and an sdesc by using different colors to show which is which. (I just picked magenta because it wasn't in use yet xD)

Reply:
Since there is a change to separate `recog`'d and un-recoged `sdesc`s now, there needs to be new unit tests to also cover the non-recog'd case. 
Also, since `RecogHandler.get()` has changed in purpose, its docstring needs to be updated' both its `Returns:` stanza as well as the general instruction on how this is to be used.